device-types/Dell/PowerEdge-FX2s

```diff
@@ -0,0 +1,35 @@
+---
+manufacturer: Dell
```

Comment:
This isn't something we need to set up right away but it might be nice to figure out a way to validate PRs in the future to make sure they are using the right name for the manufacturer. I've seen netbox instances use `Dell Inc.` or `dell` etc. Maybe just listing them in the readme is fine for now, I'm not sure how else we would automatically validate it.

Reply:
nb-dt-import.py had issues with "Dell Inc."  as the manufacturer name.  I originally used that full string because that's what redfish reports as manufacturer.  Functionally, manufacturer lookups don't come into play when working with devices, because most lookups are done on model.  I am planning on purging "Dell Inc." and just naming everything "Dell" unless you see an issue with it.